### DISCUSSION TITLE
Error suddenly

### DISCUSSION THREAD

**Discussion by zSynctic** [spyoungtech/ahk#261] [Q&A] [answered]
I used it multiple times but suddenly now when running the program I get this error:

Exception has occurred: AhkExecutableNotFoundError
Could not find AutoHotkey.exe on PATH. Provide the absolute path with the `executable_path` keyword argument or in the AHK_PATH environment variable. You can likely resolve this error simply by installing the binary extra with the following command:
	pip install "ahk[binary]"

**Comment by spyoungtech** [comment, marked as answer]
Covered in #262
